### PR TITLE
eslint-config: turn on "eslint-comments/no-unlimited-disable" rule

### PR DESCRIPTION
**Pull request by jaroslav-kubicek**
closes https://github.com/adeira/universe/issues/337

### CODE REVIEW

File: src/eslint-config-adeira/getCommonConfig.js [hunk 1]
```diff
@@ -82,5 +83,15 @@ module.exports = function getCommonConfig(rules /*: EslintConfigRules */) /*: Es
       'eslint-plugin-adeira',
       'eslint-plugin-prettier',
     ],
+
+    overrides: [
+      {
+        files: ['**/__generated__/*.graphql.js'],
+        rules: {
+          // Relay disables generated files with unlimited scope
```
Comment (mrtnzlml) [inline src/eslint-config-adeira/getCommonConfig.js]: You know we have Relay under control and we could just generate the files with correct Eslint disables? 😅

Comment (jaroslav-kubicek) [reply src/eslint-config-adeira/getCommonConfig.js]: I know, I like your way of thinking, at one point I was considering it too except I don’t know if we should do that - we would disable only what doesn’t match our rules, but user might have different config. Therefore I would suggest to have this and let that idea sink in.

File: src/eslint-config-adeira/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
 - Added `react/jsx-no-script-url` as next_version_error
+- `eslint-comments/no-unlimited-disable` rule is turned on, ignoring all `__generated__/*.graphql.js` files by default, so files autogenerated by Relay compiler are not reported. Follow [Eslint Docs](https://eslint.org/docs/user-guide/configuring#disabling-rules-only-for-a-group-of-files) on how to disable additional group of files if needed.
```
Comment (mrtnzlml) [inline src/eslint-config-adeira/CHANGELOG.md]: Can we add here the `overrides` example as well to make it easier?

Comment (jaroslav-kubicek) [reply src/eslint-config-adeira/CHANGELOG.md]: fixed!

**Pull request merged**